main.py

```diff
@@ -240,8 +240,9 @@ def executeBot(currentAccount: Account, args: argparse.Namespace):
 
             goalPoints = utils.getGoalPoints()
             goalTitle = utils.getGoalTitle()
-
-            remainingSearches = utils.getRemainingSearches()
```

Comment:
We should keep this because if `AppriseSummary.on_error` is set and searches remain

Reply:
I just read about the switch somewhere, it did seem to avoid errors. Ran twice without selenium errors. Could be coincidence

With my PR, it will only search up to 2x (so max 60 for desktop and 40 for mobile). It will not go forever. I find often the search fail because of the terms. This gives a little more robustness as it will add a few more if there is a failure. Since it pulls 2x more terms in the init(), it will not repeat existing ones (be sure to test by deleting the google_trends files). I found this allows the linear mode which goes *much* quicker and it just cycles through the base terms quicker. When I run, I get 5-10 missing searches even after 6 retries, so this just fills in those gaps without me guessing if I missed any. 

Maybe of the apprise gave an message of how many searches were successful or missing, one could just rerun. However, I think the new google trend logic will just reuse the existing terms from that day and not pull unique ones. My logic "guarantees" the 2x terms are more unique and reduces needing to rerun.

Thoughts?

Reply:
One assumption we're making is that some terms are bad versus others. It's more likely the case that a term fails because of some throttling on Microsoft's side rather than a term being bad.

Reply:
ok. for me its bad terms. I found even waiting on 7 exponential retries still fails, Where a new term works fine even on linear after a failure.

Reply:
That's probably because the sleeps were always after the search attempt (until changes I've made today). So, the last and longest of the sleeps would be after and during that time it's pretty likely that the attempt "took" on the server's side.

The logging isn't 100% trustworthy in other words.